.github/workflows/manual-deploy-testnet-l2.yml

```diff
@@ -223,9 +223,10 @@ jobs:
           inlineScript: |
             az vm open-port -g Testnet -n "${{needs.build.outputs.RESOURCE_STARTING_NAME}}-${{ matrix.host_id }}-${{ GITHUB.RUN_NUMBER }}"  --port 13000,13001,6060,6061,10000
 
-      - name: 'Wait for Obscuro sequencer to start'
+        # To overcome issues with missing dependencies on the VM we need to wait for the VM to be ready
+      - name: 'Allow time for VM initialization'
         shell: bash
-        run: if [ 0 != ${{ matrix.host_id }} ]; then echo "Sleeping while sequencer starts" & sleep 60; fi
+        run: sleep 60
```

Comment:
We might still need the `Wait for Obscuro sequencer to start` to avoid the validator issuing a request secret before the sequencer has started ?

Reply:
There is a step higher up (`Stagger VM creation`) that staggers them by 30 seconds already, I can bump that to 60 if you like?

This sleep had a bug in it until we fixed it this week so neither of them were sleeping, so I think the 30s offset must have been doing the trick.